README.CentOS7.md

```diff
@@ -0,0 +1,152 @@
+Note that the main README in LORIS assumes that LORIS is being run on Ubuntu.
+
+This Readme assumes you already understand basic UNIX, MySQL and Apache setup and
+settings. If you're not already comfortable troubleshooting sysadmin issues,
+you should not follow this guide.
+
+For further details on the install process, please see the LORIS GitHub Wiki CentOS Install page.  
+
+# System Requirements - Install dependencies
+
+Default dependencies installed by CentOS 7.x may not meet the version requirements for LORIS deployment or development:
+* MariaDB 10.3 is supported for LORIS 21.* 
+* PHP 7.2 is supported for LORIS 21.*
+
+In addition to the above, the following packages should be installed with `yum` and may also differ from the packages referenced in the main (Ubuntu) [LORIS Readme](./README.md). Detailed command examples are provided below.
+ * Apache2
+ * Node
+ * PHP Composer
+
+## Apache2
+```
+sudo yum install httpd 
+sudo service httpd start
+```
+## PHP 7.2
+```
+sudo yum install epel-release
+sudo yum install http://rpms.remirepo.net/enterprise/remi-release-7.rpm
+sudo yum install yum-utils
+sudo yum-config-manager --enable remi-php72
+sudo yum update
+sudo yum install php72
+sudo yum install php72-php-fpm php72-php-gd php72-php-json php72-php-mbstring php72-php-mysqlnd php72-php-xml php72-php-xmlrpc php72-php-opcache php72-php-pdo php72-php-mysql
+```
+## MariaDB
+
+MySQL is not recommended/supported on CentOS, see [paragraph 2 here](https://www.digitalocean.com/community/tutorials/how-to-install-mysql-on-centos-7) for background). MySQL and MariaDB are nearly identical in operation.
+
+*Note:* LORIS developers (those NOT working with a .zip release codebase) should skip steps relating to hosting their database locally. Contact your sysadmin for database credentials.
+
+```
+sudo yum install MariaDB-client MariaDB-server
+```
+
+By default in CentOS 7, the MariaDB version is (currently) 10.2.27
+Check what version you have installed by running:
+```
+mysql -v
+```
+
+Upgrade your MariaDB to version 10.3 (LORIS 21 does not support MariaDB 10.2.27)
+Then, 
+```
+sudo service mariadb start
+sudo service mariadb status
+```
+
+To finalise the MySQL/MariaDB installation: 
+```
+mysql_secure_installation
+```
+Then follow instructions to create a password for the root user.
+
+## NodeJS
+```
+sudo yum install nodejs
+```
+
+## PHP Composer
+```
+sudo curl -sS https://getcomposer.org/installer | php
+sudo mv composer.phar /usr/local/bin/composer
+```
+# Get the Code
+
+Download the latest release from the [releases page](https://github.com/aces/Loris/releases) to the home directory (~/), unzip it, and copy the contents to your project directory, `/var/www/loris` (we recommend naming your project directory `loris`, although you can use a different naming convention if you prefer). 
+```
+wget https://github.com/aces/Loris/archive/v$VERSION.zip
```

Comment:
```suggestion
curl -s https://api.github.com/repos/aces/loris/releases/latest \
| grep 'tarball' \
| cut -d : -f2,3 \
| tr -d \" \
| tr -d , \
| wget -i -
```

This command will use the GitHub API to always get the latest release. It might be useful but you don't have to include it.